src/cookieConsentManager.js

```
@@ -0,0 +1,128 @@
+    } else {
+      // Exact match removal
+      document.cookie = `${name}=; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/; domain=${location.hostname}`;
+    }
+  });
+};
+
+function removeGoogleAnalyticsScriptsAndCookies() {
+  removeScript(`script-${this.name}`);
+  removeCookies(this.cookies);
+  window.dataLayer = undefined; // Clear global state
+  delete window.gaGlobal;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)